editor/plugins/tiles/tile_atlas_view.cpp

```diff
@@ -448,11 +448,17 @@ void TileAtlasView::_draw_alternatives() {
 }
 
 void TileAtlasView::_draw_background_left() {
-	background_left->draw_texture_rect(theme_cache.checkerboard, Rect2(Vector2(), background_left->get_size()), true);
+	bool draw_checkerboard = EDITOR_GET("editors/tiles_editor/draw_checkerboard");
```

Comment:
This should be fetched once at startup and then when settings change (with the `NOTIFICATION_EDITOR_SETTINGS_CHANGED` notification) instead of being loaded every time we draw

Reply:
Unsure if I have implemented it correctly but I have tried to do as suggested. Now is grabbed at start up and then when setting is changed, also fixed an issue where the UI was not redrawn to show the change.